lib/XMLHttpRequest.js

```diff
@@ -504,7 +538,7 @@ exports.XMLHttpRequest = function () {
    */
   this.handleError = function (error) {
     this.status = 0
-    this.statusText = error
+    this.statusText = JSON.stringify(error)
```

Comment:
This is less wrong but still wrong, see #30 
Could you please revert this change? I've addressed this issue in #31.
Ideally, we should infer the status text from the error but that would require a bit of work. The reason is that we would have to generate the exhaustive list of all the errors that might happen :disappointed:

Reply:
I think #31 isn't quite right either, I've addressed this in a new commit.

Reply:
BTW, this was previously untested, and is now tested in the last `test-request` test in 
```
        expect(responseText).to.equal('')
        expect(errorText).to.equal('{"code":"CERT_HAS_EXPIRED"}')
```